src/bundle/DependencyInjection/Compiler/ComponentPass.php

```diff
@@ -46,7 +51,13 @@ public function process(ContainerBuilder $container): void
                 }
 
                 $id = $tag['id'] ?? $id;
-                $registryDefinition->addMethodCall('addComponent', [$tag['group'], $id, $serviceReference]);
+                $group = $tag['group'];
+
+                //adding prefix to align with new name convention related to new twig-components package
+                if (!str_starts_with($group, self::GROUP_PREFIX)) {
+                    $group = self::GROUP_PREFIX . $group;
+                }
```

Comment:
This changes the behavior of the group rendering, and is quite hidden within the compiler pass.

Going from older 4.6 into newer 4.6, you need to know that whenever you called group rendering without the prefix, it will now be present ONLY with the prefix. So old calls without prefix will not render anything.

Given that this functionality is documented, this will mean we are breaking our contract.

This requires either a different approach, or a lot more visibility at least.